driver/config/.schema/config.schema.json

```diff
@@ -355,7 +380,6 @@
         "id",
         "provider",
         "client_id",
-        "client_secret",
```

Comment:
client secret is not required for apple. So I'm adding it as a required field in a condition below